app/assets/stylesheets/issues.scss

```diff
@@ -0,0 +1,8 @@
+// Place all the styles related to the Issues controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+.simple_form.Issue {
```

Comment:
Selector `simple_form` should be written in lowercase with hyphens<br>Selector `Issue` should be written in lowercase with hyphens